weblab/datasets/tests/test_views.py

```diff
@@ -465,7 +465,7 @@ def test_uses_octet_stream_for_unknown_file_type(self, mock_guess, my_dataset_wi
         )
 
         assert response.status_code == 200
-        assert response['Content-Type'] == 'application/octet-stream'
+        assert response['Content-Type'] == 'text/csv'
```

Comment:
Change the file extension on line 464 instead - this test is supposed to check a unrecognised file type!